### ISSUE TITLE
Fix tabbing issues

### ISSUE THREAD

**Issue by superhighfives** [superhighfives/pika#18]
**Describe the bug**

Looks like tabbing is the wrong order, and doesn't encompass all the items. Should be keyboard navigable.

**To Reproduce**
1. Open the app in Catalina 10.15 with keyboard tabbing enabled (screenshot below)
2. Blue hover will appear on half the colour details button on the right side
3. Tabbing from there is out of order and doesn't include all fields / buttons.

**Expected behavior**
Tabbing should be clear and in an expected order.

**Screenshots**

![Screenshot of Pika with blue halo on the right side colour picker dropdown button](https://user-images.githubusercontent.com/449385/105159776-60b05680-5b07-11eb-9102-7b0dd6e863c6.png)

![Screenshot of preferences with keyboard navigation enabled](https://user-images.githubusercontent.com/449385/105160010-a3722e80-5b07-11eb-8531-2d4b0b1de499.jpg)

**Environment (please complete the following information):**
 - OS: macOS Catalina
 - App Version: All versions

**Additional context**
Thanks for flagging, @geelen.

**Comment by superhighfives** [comment]
Currently I'm supressing the focus ring in #13, which is not a great accessibility experience, until I can take the time to overhaul the accessibility in general.
